inc/cachify.class.php

```diff
@@ -1124,6 +1086,80 @@ private static function _is_logged_in() {
 		return false;
 	}
 
+	/**
+	 * Register all hooks to flush the total cache
+	 *
+	 * @since   2.4.0  Make the function public
+	 *
+	 * @param   void
+	 * @return  void
+	 */
+	private static function register_flush_cache_hooks(){
```

Comment:
Please, make the method `public`. Although it is private by logic, it must be public when executed from within a hook (see also my other comment).